Formula/g/gap.rb

```diff
@@ -0,0 +1,38 @@
+class Gap < Formula
+  desc "Groups, Algorithms, Programming - A system for computational discrete algebra"
+  homepage "https://www.gap-system.org"
+  url "https://github.com/gap-system/gap/releases/download/v4.13.0/gap-4.13.0.tar.gz"
+  sha256 "cc76ecbe33d6719450a593e613fb87e9e4247faa876f632dd0f97c398f92265d"
+  license "GPL-2.0-or-later"
+
+  depends_on "autoconf" => :build
+  depends_on "libtool" => :build
+  depends_on "gmp"
+  depends_on :macos
+  depends_on "readline"
+  depends_on :xcode
+  uses_from_macos "zlib"
+
+  def install
+    system "./configure",
+      "--prefix=#{prefix}",
+      "--with-gmp=#{Formula["gmp"].include}",
+      "--with-readline=#{Formula["readline"].include}"
+    system "make", "install"
+    pkgshare.install "pkg"
```

Comment:
What is this for?

Reply:
1. For whatever reason gap tries to use the system gap and readline so they need to be specified.
2. GAP needs to run make install in order to use prefix etc
3. (presumably this is what you meant) GAP builds packages in the pkg folder but doesn't "install" them with make install so they need to be manually installed into pkgshare

Reply:
Yeah, only meant 3 but GitHub quotes some earlier lines too. Can you ask upstream to rectify this missing part of `make install`?

Reply:
I can ask them but I doubt they'll change that unless I take over the work. I will mention this to them.